src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -8,7 +8,7 @@
 // associated with an assembly.
 [assembly: AssemblyCompany("Autodesk, Inc")]
 [assembly: AssemblyProduct("Dynamo")]
-[assembly: AssemblyCopyright("Copyright © Autodesk, Inc 2023")]
+[assembly: AssemblyCopyright("Copyright � Autodesk, Inc 2023")]
```

Comment:
no reason to commit this file unless you are fixing some line endings or something?

Reply:
@Cristhian-Marin-asdk is this change required? Otherwise please revert this before we merge.

Reply:
done